shared/crypto/input/index.tsx

```diff
@@ -176,17 +189,117 @@ export const FileInput = (props: FileProps) => {
   )
 }
 
-export const OperationBanner = React.memo((props: OperationBannerProps) => {
-  const color = props.type === 'error' ? 'red' : props.type === 'warning' ? 'yellow' : 'grey'
+export const Input = (props: InputProps) => {
+  const {fileDroppedCounter, operation} = props
+  const dispatch = Container.useDispatch()
+
+  // Store
```

Comment:
i know we have these comments in some components but i don't think it actually makes things more clear, just makes the component larger

Reply:
Happy to remove. I find it visually organizes things much in the same way that `mapStateToProps` and `mapDispatchToProps` organize state selection and dispatching in a connected component.